tests/run_tests.py

```diff
@@ -446,6 +446,12 @@ def test_fuzzy_patch_after(self):
         self.assertTrue(pto.apply(root=treeroot, fuzz=True))
         self.assertFalse(pto.apply(root=treeroot, fuzz=False))
 
+    def test_unlink_backup_windows(self):
+        treeroot = join(self.tmpdir, 'rootparent')
+        shutil.copytree(join(TESTS, '11permission'), treeroot)
```

Comment:
Can we explicitly modify the permissions here? At least we see the intention although we are not running tests in Windows here

Reply:
Yes, sure. Letme update it

Reply:
done, now we are forcing to read-only. The patch will be applied over some_file which is read-only.